resources/views/admin/categories.blade.php

```diff
@@ -104,6 +122,20 @@
                                                         </td>
                                                         <td>{{$sub->mainid->name}}</td>
                                                         <td>{{$sub->slug}}</td>
+                                                        <td>
```

Comment:
Also in the view, it is almost same. I think we could easily make a partial view.